types/gorilla-engine/index.d.ts

```diff
@@ -381,8 +383,8 @@ declare namespace GorillaEngine {
         title: string;
         message: string;
         iconType: "info" | "question" | "warning";
-    }): void;
-    function calculateTextWidth(text: string, font: string, fontSize: number, fontKerning: number): number;
+    }): Promise<any>;
```

Comment:
I can’t seem to find this function in the documentation, but if it’s an async function with no return then the proper type is `Promise<void>`:

```suggestion
    }): Promise<void>;
```

If there’s a specific type that this returns (i.e. `number`), then that should be used:

```suggestion
    }): Promise<number>;
```